### DISCUSSION TITLE
What's useful for people leaving?

### DISCUSSION THREAD

**Discussion by mattl** [foocorp/hacienda#93] [General]
What's a useful export of data look like for people leaving Libre.fm?

**Comment by pluc** [comment]
An export is only useful depending on where you want to export it, i'd say. So whatever format libre.fm accepts, don't know if last.fm accepts bulk submission. If we're talking purely to have the data, then probably JSON would be a good standard (albeit quite voluminous for 100k scrobbles),

One thing that could be cool but wildly unnecessary is a script that attaches your scrobbles to tracks' ID3. Play count, URL.. can't think of much else

**Comment by brianredbeard** [comment]
While boring and problematic because of UTF characters, a simple field delimited row based text format (e.g. CSV, TSV, etc) as a lowest common denominator.  This allows users to still efficiently process their data with the POSIX toolkit (grep, awk, sed, cut, col, etc).

To be clear, I don't think this should be the *only* option... but it certainly make the produced data accessible on a wide variety of systems without additional tooling.
